makefiles/shared.mk

```diff
@@ -72,5 +72,8 @@ next-gen-stage: ## Host online for review
 		mut-publish public ${BUCKET} --prefix="${MUT_PREFIX}" --stage ${ARGS}; \
 		echo "Hosted at ${URL}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/"; \
 	fi
+
+oas-page-build:
```

Comment:
We'll want to add `next-gen-parse` as a prerequisite, similar to the `next-gen-html: next-gen-parse` target - and we'll want to move this target definition up to line 64, to maintain order of execution 😄

Reply:
Ah, I intended `make oas-page-build` to be called as a build command by the autobuilder after `next-gen-html` for [staging jobs](https://github.com/mongodb/docs-worker-pool/blob/42bd36b1f52e49c646a79474c12d299f33d774cb/src/job/stagingJobHandler.ts#L57) and [production jobs](https://github.com/mongodb/docs-worker-pool/blob/1a482242fa6f1463abb059884cddb2c56ba9fad9/src/job/productionJobHandler.ts#L68) so that it can be done after frontend files are built, but before those files are uploaded by mut.

Is having prerequisites for `oas-page-build` preferred?

Reply:
Not strictly preferred and I understand how this is meant to work a bit better now - only nit I have is to add a comment to the makefile that references the lines you've linked 😄

Reply:
I definitely should have added that as a note in the description. I'm sorry about the lack of context there!

Comment has been added. I think a follow-up can be done to have the links point to the actual lines that the command is invoked instead of the `make next-gen-html` command